### PR TITLE
Trigger Pipeline on Pushing Tags

### LINKED ISSUE #16: Explicitly State Tags in CI Pipeline
When pushing a tag then the CI pipeline is not triggered. We were under the assumption that this should work since it is a push, too.

To fix this we need to configure the pipeline so that any tag having a prefix of `v` on branch `main` will trigger the pipeline.

```
on:
  push:
    branches:
    - main
    tags:
    - v**
```

This should do the trick.

### PR DESCRIPTION
**Pull request by DiCanio**
Fixes #16.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
     - main
+    tags:
+    - v**
```
Comment (EmteZogaf) [inline .github/workflows/ci.yml]: Shouldn't matching a semantic version be more favorable? Something like `v[0-9]+.[0-9]+.[0-9]+**`, which matches for example  `v1.12.199` or `v56.1.0-alpha1+x86_64`?

**Review by EmteZogaf — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (EmteZogaf) [reply .github/workflows/ci.yml]: I just build the pattern according to the documentation: [Filter Pattern Cheat Sheet](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), but didn't test.

Comment (DiCanio) [reply .github/workflows/ci.yml]: I don't see a problem switching from my pragmatic approach to the one you proposed. Seems to be more fitting.
👍🏻

**Review by EmteZogaf — APPROVED** [review]
lgtm

**Pull request merged**